chapters/glossary/glossary.rst

```diff
@@ -0,0 +1,78 @@
+@@ -1,75 +0,0 @@
+Glossary
+============
+
+The following terminology is commonly used within the Jina framework. Jina is an active development, meaning these terms may be updated and refined regularly as new and improved features are added.
+
+.. glossary::
+
+    Chunk
+        Chunks are individual semantic units of a Document. This occurs within the Segmenter class of Executors. Examples of individual units would be sentences from large documents or pixel patches from an image. `For further information see the Understand Jina Recursive Document Representation guide. <https://docs.jina.ai/chapters/traversal/index.html?highlight=recursive>`_
+    Client
+        A Python client that connects to a remote Jina gateway.
+    Container
+        A Docker container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings.
+    Crafter
+        Crafters are a class of Executor whose purpose is to alter a Document to match a certain requirement. This could include tasks such as resizing images or reducing all text in a document to lowercase.
+    Document
+        Documents are the most basic data types available within Jina. Anything you want to search for within Jina is considered a document, this could include images, sounds clips or text documents.
```

Comment:
```suggestion
        Documents are the most basic data types available within Jina. Anything you want to search for within Jina is considered a document. This could include images, sounds clips or text documents.
```

Run-on sentence. Could be shortened as so

Reply:
I am not sure most basic is the right wording for such an important concept

Reply:
Fundamental, central?

Reply:
yes something in this direction